Description: Prepare Makefile for porting the modules
 Remove all targets from obj-y to prepare to port the kernel modules
 in sequence. Define IFS_DEB9 instead of IFS_RH74.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: not-needed
Last-Update: <2019-01-31>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -7,8 +7,8 @@
 ifneq ($(KERNELRELEASE),)
 #kbuild part of makefile
 
-CFLAGS_MODULE += -DUSE_PI_LED_ENABLE=1 -DIFS_RH76
-obj-y := rdmavt/ hfi1/ ib_qib/ ib_ipoib/
+CFLAGS_MODULE += -DUSE_PI_LED_ENABLE=1 -DIFS_DEB9
+obj-y :=
 
 else
 #normal makefile
